udp-server/mock_find_location.py

```diff
@@ -8,10 +9,10 @@ def __init__(self, tag_ids):
 		"""
 		self.player_dict = {
 			tag_ids[0]: TagPosition(1, 1),
-			tag_ids[1]: TagPosition(100, 100),
-			tag_ids[2]: TagPosition(550, 200),
-			tag_ids[3]: TagPosition(50, 50),
-			tag_ids[4]: TagPosition(700, 200),
+			#tag_ids[1]: TagPosition(100, 100),
```

Comment:
regner vi med det skal bruges igen? hvis ikke skal det vel bare fjernes i stedet for at udkommenteres? 🤔

Reply:
Altså denne klasse bliver udelukkende brugt til når vi tester, så tænker vel ikke det gør så meget at der er udkommenteret kode

Reply:
i guess 🤔